aw_watcher_terminal/main.py

```diff
@@ -71,9 +90,15 @@ def init_client() -> None:
         config.logger.info("Created bucket: {}".format(bucket_id))
 
 
+def clean_up():
+    config.logger.info('Doing cleanup')
+    if config.client:
+        config.client.disconnect()
```

Comment:
Why not just move this to after waiting for the futures here?
https://github.com/Otto-AA/aw-watcher-terminal/pull/11/files#diff-d5965acd7e355de88f7da51dbe93977cR58

Reply:
My assumption was, that if the process is killed, it wouldn't execute then. But apparently it doesn't do that with `atexit` either. So if you don't think that it is that important, I can move it there, else I can catch kill signals like this:
```python
atexit.register(clean_up)
signal.signal(signal.SIGTERM, clean_up)
signal.signal(signal.SIGINT, clean_up)
```

Reply:
Might work, but if not it's not worth it to spend time on these minor things. Just skip disconnect() altogether and let there be a possibility that a last second event to be dropped, it's not the end of the world.